proto/tendermint/types/header.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
```

Comment:
Won't this complicate backporting?

Reply:
Yes, it will complicate backporting. However, I didn't find a better way of doing this, as we are getting into some dependency loops (both types.Validator and abci.ValidatorUpdate need to use the NetworkEndpoint struct)

Reply:
@QuantumExplorer @iammadab your thoughts?

Reply:
Awesome!